.eslintrc.json

```
@@ -0,0 +1,35 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.test.js", "**/__tests__/*.test.js", "**/*.cy.js"],
```

```suggestion
      "files": ["**/*.test.js", "**/__tests__/*.test.js"],
```

We use different lint rules for jest and cypress, you should have two separate blocks here with two overrides.